packages/teams-js/src/public/teamsAPIs.ts

```diff
@@ -46,12 +46,14 @@ export namespace teamsCore {
   }
 
   /**
-   * @hidden
    * Registers a handler to be called when the page has been requested to load.
    *
+   * @see Check out [App Caching in Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/build-tabs-for-meeting?tabs=desktop%2Cmeeting-chat-view-desktop%2Cmeeting-stage-view-desktop%2Cchannel-meeting-desktop#app-caching)
```

Comment:
I'm not sure if the Typedoc `@see` tag is supported by MS Docs build system--it might be stripped out upon publish. But I guess we will [at]see . . . :-p

Reply:
@erikadoyle  is correct. You can look at the `OpenSingleChatRequest` interface in teams-js/src/public/chat.ts:

https://learn.microsoft.com/en-us/javascript/api/@microsoft/teams-js/opensinglechatrequest?view=msteams-client-js-latest

The link gets stripped out

Reply:
Is there a different way to link to the doc we can use? Maybe @remarks?

Reply:
I think I can keep the link in and remove @see and it'll work. Not sure how to test it because both work fine when looking at the generated tsdocs in the repo

Reply:
![image](https://user-images.githubusercontent.com/112656629/216151916-9f82fcb1-c138-472d-bbae-d0ae6b8c1ad9.png)

This is how it looks on my end

Reply:
FWIW there is a different system (MS Docs) that consumes typedoc tags to render the reference documentation, so sometimes there are typedoc tags that aren't supported by their system (even though it looks correct to you). @erikadoyle please correct me if I'm wrong!